code/modules/recycling/disposal.dm

```diff
@@ -431,6 +431,9 @@
 	var/turf/target
 	playsound(src, 'sound/machines/hiss.ogg', 50, 0, 0)
 	if(H) // Somehow, someone managed to flush a window which broke mid-transit and caused the disposal to go in an infinite loop trying to expel null, hopefully this fixes it
+		if(H.destinationTag)
+			playsound(src, 'sound/misc/yougotmail.wav', 50, 0, 0)
+			visible_message("<span class='notice'><font size=4><i>You've got mail!</i></font></span>")
```

Comment:
This message might be more clear if you add `[bicon(src)]` to the beginning of the message, so that the "you've got mail" message shows a little disposal icon